### PR TITLE
03 git solution

### PR DESCRIPTION
**Pull request by serhii-siedin**
Completed the assignment according to the requirements for lesson 3

### CODE REVIEW

**Comment by alexposukhov** [comment]
Please rename pull request description according to current task

File: 03_module/hello_mod.c [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#include <linux/module.h>
+#include <linux/moduleparam.h>
+#include <linux/init.h>
+#include <linux/kernel.h>
+
+static int a=10;
```
Comment (alexposukhov) [inline 03_module/hello_mod.c:6]: Please set initial value to 0

File: 03_module/hello_mod.c [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+module_param(a,int,0660);
+
+static int b=15;
```
Comment (alexposukhov) [inline 03_module/hello_mod.c:9]: Same here

**Review by alexposukhov — CHANGES_REQUESTED** [review]
Please use checkpatch.pl script to fix warnings and errors.

**Comment by alexposukhov** [comment]
Please note that parameter should be transferred to module during loading:
`$insmod hello_mod.ko a=5 b= 7`

**Review by SyedinSN — COMMENTED** [review]
fulfilled all the indicated comments

**Comment by SyedinSN** [comment]
Changes according to the comments were made in branch 03_module_solution

**Pull request closed without merging**